src/components/Tag/Tag.tsx

```diff
@@ -28,24 +27,19 @@ export type TagProps = NonMutuallyExclusiveProps &
 
 export function Tag({children, disabled = false, onClick, onRemove}: TagProps) {
   const i18n = useI18n();
-  const {newDesignLanguage} = useFeatures();
 
   const className = classNames(
     styles.Tag,
     disabled && styles.disabled,
     onClick && styles.clickable,
     onRemove && styles.removable,
-    newDesignLanguage && styles.newDesignLanguage,
   );
 
   const ariaLabel = i18n.translate('Polaris.Tag.ariaLabel', {
     children: children || '',
   });
 
-  const buttonClassName = classNames(
-    styles.Button,
-    newDesignLanguage && styles.newDesignLanguage,
-  );
+  const buttonClassName = classNames(styles.Button);
```

Comment:
We don't need to use the classNames function here anymore. Just put the style directly on the element